CloudWatch2S3.template

```diff
@@ -147,13 +147,17 @@ Outputs:
             - LogBucket
             - Arn
         - Ref: BucketName
+    Export:
+      Name: !Sub '${AWS::Region}-${AWS::StackName}-BucketArn'
   LogDestination:
     Description: Log destination ARN to be used when setting up other accounts to
       exports logs
     Value:
       Fn::GetAtt:
         - LogDestination
         - Arn
+    Export:
+      Name: !Sub '${AWS::Region}-${AWS::StackName}-LogDestinationArn'
```

Comment:
Do we need the region here? Export names should be unique per-region.

Reply:
yes, you are correct.  will remove region.

Reply:
done